jupyter_server/event_schemas/kernels/v1.yaml

```diff
@@ -0,0 +1,50 @@
+"$id": https://events.jupyter.org/jupyter_server/kernels/v1
+version: 1
+title: Kernel Manager activities
+personal-data: true
+description: |
+  Record events of a kernel manager.
+type: object
+required:
+  - action
+  - kernel
+  - message
+properties:
+  action:
+    enum:
+      - start
+      - interrupt
+      - shutdown
+      - restart
+      - error
+    description: |
+      Action performed by the Kernel Manager.
+
+      This is a required field.
+
+      Possible values:
+
+      1. start
+         A kernel has been started with the given kernel id.
+
+      2. interrupt
+         A kernel has been interrupted for the given kernel id.
+
+      3. shutdown
+         A kernel has been shut down for the given kernel id.
+
+      4. restart
+         A kernel has been restarted for the given kernel id.
+
+      5. error
+         A kernel has been errored out.
+  kernel:
```

Comment:
We should probably rename this to `kernel_id` so that it's less ambiguous since "kernel" is commonly used to refer to the kernel name.  (Do we also need `kernel_name`?  We'd need to check that that information is available across the actions.)

Reply:
Added both, lets talk discuss the pattern.

Reply:
I think `kernel_name` should also be emitted.